src/ppx/ppx_eliom_utils.ml

```diff
@@ -461,8 +469,277 @@ end
      ]
    ]
 *)
-module Shared = struct
+module Rpc = struct
+  type error = No_arguments | Missing_argument_type | Fatal_error
+
+  let print_error ~loc (e : error) =
+    let string_of_error =
```

Comment:
usually `string_of_something` are functions translating `something` to string in OCaml standards. `error_str` or something silimilare would fit a bit more to what you have have here.